src/ir/attrs.cc

```diff
@@ -53,9 +53,13 @@ Array<AttrFieldInfo> DictAttrsNode::ListFieldInfo() const {
   return {};
 }
 
-DictAttrs::DictAttrs(Map<std::string, ObjectRef> dict) {
+DictAttrs::DictAttrs(Map<String, ObjectRef> dict) {
   ObjectPtr<DictAttrsNode> n = make_object<DictAttrsNode>();
-  n->dict = std::move(dict);
+  Map<std::string, ObjectRef> dict_copy;
+  for (auto iter : dict) {
```

Comment:
The additional copy is certainly quite costly, leave Map<std::string, ObjectRef> as it is for now, before we migrated the container

Reply:
@tqchen : You are right! It was an effort to isolate independent commits, so that the dependency can be broken down. Will revert it for now!